cmsController.php

```diff
@@ -75,10 +75,10 @@ function ArticleList ($db) {
  * Takes selected article POSTdata and selects values from DB based on POST value
  *
  * @param $db to select from
- * @param $postData user response for article to select
+ * @param assoc array $postData user response for article to select
```

Comment:
switch data type and variable around for doc block.

Reply:
resolved